common/pb/response_adjust_mod.proto

```diff
@@ -0,0 +1,7 @@
+import "response.proto";
+
+message Response_AdjustMod{
+	extend Response {
+		optional Response_AdjustMod ext = 1011;
```

Comment:
FYI, i did find a fix for the proto warnings so Ill throw a PR in to fix the formatting as well as the warnings all at once.